snare/cloner.py

```diff
@@ -127,6 +128,7 @@ def _make_filename(self, url):
             self.visited_urls.append(current_url.human_repr())
             file_name, hash_name = self._make_filename(current_url)
             print('name: ', file_name)
```

Comment:
Since you want to move everything into logs, maybe it's better to remove print?

Reply:
Yup.